### PR TITLE
Add blog post on media appearances

### PR DESCRIPTION
**Pull request by astrojuanlu**
Ready for review.

Rendered version: https://readthedocs-blog--112.org.readthedocs.build/behind-the-docs-1/

### CODE REVIEW

**Comment by astrojuanlu** [comment]
(Also, in line with https://github.com/readthedocs/readthedocs.org/issues/7327#issuecomment-831440310, I used this as an experiment to rename `master` to `main`. At least the PR builder work without any changes 👍🏽 let's see what happens when we merge)

File: behind-the-docs-1.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   :tags: behind-the-docs
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Learn more about Read the Docs directly from its team members:
+      appearances in the media during the first half of 2021.
+
+Behind the Docs episode 1
```
Comment (ericholscher) [inline behind-the-docs-1.rst:11]: Don't love this branding -- We should at least say `Behind the Scenes at Read the Docs` -- I want to get away from cutesy branding, and this definitely won't be obvious to users what it is :)

```suggestion
Behind the Scenes at Read the Docs
```

For something with more SEO value:

```suggestion
Early 2021 talks and podcasts from Read the Docs
```

File: behind-the-docs-1.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Watch Eric online:
+
+.. image:: /img/eric-upstream.png
+   :align: center
+   :target: `"Bootstrapping a sustainable open source project: The Read the Docs story"`_
+
+.. _`"Bootstrapping a sustainable open source project: The Read the Docs story"`: https://explore.tidelift.com/upstream/main/session-eric-holscher
+
+"Making documentation sexy" @ Data for Future podcast
```
Comment (ericholscher) [inline behind-the-docs-1.rst:60]: I don't love this title -- I'd like to keep `sexy` out of anything we're promoting. Definitely fraught and could be seen in a negative light. Probably use `exciting` or something more explicit and descriptive, if possible.

File: behind-the-docs-1.rst [hunk 3]
```diff
@@ -0,0 +1,89 @@
+.. post:: July 1, 2021
+   :tags: behind-the-docs
```
Comment (ericholscher) [inline behind-the-docs-1.rst:2]:
```suggestion
   :tags: behind-the-scenes, media, podcast, talk```

File: behind-the-docs-1.rst [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+.. image:: /img/eric-upstream.png
+   :align: center
+   :target: `"Bootstrapping a sustainable open source project: The Read the Docs story"`_
+
+.. _`"Bootstrapping a sustainable open source project: The Read the Docs story"`: https://explore.tidelift.com/upstream/main/session-eric-holscher
+
+"Making documentation sexy" @ Data for Future podcast
+-----------------------------------------------------
+
+Also in June, `Juan Luis`_ was invited for the second time
```
Comment (ericholscher) [inline behind-the-docs-1.rst:63]: I'd put this 2nd in the list, and my upstream talk last, so it feels more balanced.

File: behind-the-docs-1.rst [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+to `Episode #39 of Data for Future`_,
+a podcast that seeks to "inspire positive actions
+towards an environmentally sustainable and socially just future through technology".
+In an informal conversation,
+Juan Luis shared some numbers about the state of software documentation according to several surveys,
+promoted the `Diátaxis <https://diataxis.fr/>`_ documentation framework created by Daniele Procida,
```
Comment (ericholscher) [inline behind-the-docs-1.rst:69]: ❤️

**Review by ericholscher — APPROVED** [review]
Great overview. I think we mostly need to think about the goals of this content and who we're trying to reach to know what the right title is.

File: behind-the-docs-1.rst (hunk 1 above)
Comment (astrojuanlu) [reply behind-the-docs-1.rst:11]: I knew you would _love_ it 😄 Accepting your second suggestion!

File: behind-the-docs-1.rst (hunk 4 above)
Comment (astrojuanlu) [reply behind-the-docs-1.rst:63]: Right, I was using chronological order, but sounds good to me 👍🏽

File: behind-the-docs-1.rst (hunk 2 above)
Comment (astrojuanlu) [reply behind-the-docs-1.rst:60]: Yeah, didn't choose the title of the podcast (different audience). Will just remove it from our heading.

**Review by humitos — APPROVED** [review]
I liked the summaries of each audio episode.

File: behind-the-docs-1.rst [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+      appearances in the media during the first half of 2021.
+
+Behind the Docs episode 1
+=========================
+
+During the first half of 2021 several team members of Read the Docs
+have talked about the company in podcasts and online events.
+If you want to learn more about our company and the work we do
+directly from the people that are a core part of it,
+and hear our thoughts on other topics
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply behind-the-docs-1.rst:18]: removed "and the work we do", kind of redundant

File: behind-the-docs-1.rst [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+like open source sustainability and developer advocacy,
+this blog post is for you.
+
+Eric @ Sustain podcast
+----------------------
+
+`Eric`_ was the special guest of `Episode 66 of Sustain`_ back in February,
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply behind-the-docs-1.rst]: Done!

**Comment by astrojuanlu** [comment]
Resolved nits, automerging 👍🏽 Thanks both!

**Comment by astrojuanlu** [comment]
Changed the title but not the filename. Will send a quick PR with that.

**Pull request merged**